paper/paper.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Becquerel: a Python package for analyzing nuclear spectroscopic measurements'
+tags:
+  - Python
+  - nuclear engineering
+  - spectroscopy
+  - calibration
+  - peak fitting
+  - nuclear data
+authors:
+  - name: Jayson R. Vavrek
+    affiliation: 1
+  - name: Mark S. Bandstra
+    affiliation: 1
+  - name: Brian Plimley
+    affiliation: 1
+  - name: Joseph C. Curtis
+    affiliation: 1
+  - name: Marco Salathe
+    affiliation: 1
+  - name: Chun Ho Chow
+    affiliation: 1
+  - name: Tenzing H.Y. Joshi
+    affiliation: 1
+  - name: J. Ali Hanks
+    affiliation: 2
+affiliations:
+  - name: Applied Nuclear Physics group, Lawrence Berkeley National Laboratory
+    index: 1
+  - name: Department of Nuclear Engineering, University of California, Berkeley
+    index: 2
+date: 04 Mar 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Nuclear spectroscopic analysis follows a typical workflow: collect binmode or listmode spectra of radiation emissions, apply a calibration from detector observables to energy deposition, fit spectral peaks to determine the number of counts detected above background, and then relate changes in net counts to measurement parameters. While there are several open-source general scientific computing packages such as ``ROOT`` and ``numpy`` suitable for handling spectroscopic data, no dedicated solution exists for both managing and analyzing spectroscopic measurements, forcing students and researchers to develop their own codes independently.
+
+``becquerel`` is a Python package for analyzing nuclear spectroscopic measurements that seeks to prevent this wide duplication of efforts. It provides open-source standard analysis tools, including peak finding and fitting, automated energy calibrations, and file I/O across several widely-used formats in nuclear spectroscopy. Built atop the Python scientific stack of ``numpy``, ``scipy``, ``pandas``, and ``numba``, ``becquerel`` is fast, flexible, and easy to use given even an introductory knowledge of Python. In addition, ``becquerel`` provides a comprehensive test suite, coverage metrics, and several example notebooks for quickly getting started with various analyses. Finally, it also provides a convenient Python interface to the NIST XCOM [@xcom] and NNDC [@nndc] nuclear databases, eliminating the need for ad-hoc downloads or manual data entry.
```

Comment:
I would add rebinning (i.e. change counts to a different set of bin edges) here, which is the most used feature from my side so far.